### DISCUSSION TITLE
Changed password on Bluelink-account

### DISCUSSION THREAD

**Discussion by Aber-s** [Hyundai-Kia-Connect/kia_uvo#312] [Q&A] [answered]
I've recently changed my password for my account on the Bluelink app and after that, the integration with Home Assistant stopped working. Although that's quite obvious, my question is how to change my password within the integration? I haven't found anything regarding this topic in the readme.

Thanks in advance!

**Comment by cdnninja** [comment, marked as answer]
Right now you need to remove and readd the integration.

**Reply by Aber-s** [reply]
Thanks, that did the job! Good to know: all my (custom) cards, scripts and automations all started working again and didn't have to be reinstalled of remade. Allthough I made a back-up, it wasn't neccessary. So it's actually quite ease.
